templates/credentials/shop/thank-you.html

```diff
@@ -0,0 +1,37 @@
+{% extends "credentials/base_cred.html" %}
+
+{% block title %}Canonical Credentials -- Your exams{% endblock %}
+
+{% block meta_description %}The Canonical Ubuntu Essentials exams certify knowledge and verify skills in general Linux,
+Ubuntu Desktop, and Ubuntu Server topics.{% endblock meta_description %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1QuhO-9FEOGLrYp8bErS_9snqdljl7d6tFAUoNQxoVDQ/edit{% endblock
+meta_copydoc %}
+
+{% block content %}
+<section class="u-fixed-width p-strip">
+  <div class="row">
+    <h2>Thank You for your order!</h2>
+    <p class="p-heading-4">
+      {{ quantity }} x {{ product }}
+    </p>
+  </div>
+  <div class="row">
+    <div class="col-8">
+      <h3>Manage Your Exams</h3>
+      <p>
+        Need to schedule or re-schedule an exam? Go to
+        /credentials/your-exams.
```

Comment:
We could just replace the `/credentials/exams` with the actual link on line 25.

Reply:
I changed the text to `Go to your exams below.`, since there is already a `Your Exams` button